app/src/main/java/org/traccar/manager/DevicesActivity.java

```diff
@@ -15,13 +15,16 @@
  */
 package org.traccar.manager;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 
 public class DevicesActivity extends AppCompatActivity {
 
+    private DevicesFragment devicesFragment;
```

Comment:
I'm not testing it. I'm looking at the code and I see that `devicesFragment` is never initialized if activity is re-created.